app/src/main/java/com/github/polypoly/app/menu/RankingsActivity.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.polypoly.app.menu
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Tab
+import androidx.compose.material.TabRow
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.text.font.FontWeight
+
+class RankingsActivity : MenuActivity("Rankings") {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            MenuContent {
+                RankingsContent()
+            }
+        }
+    }
+
+    /**
+     * The rankings view, composed of a row of rankings lists tabs
+     */
+    @Composable
+    private fun RankingsContent() {
+        // TODO: Retrieve all registered users from the database
+
+        val tabs = listOf("Most Wins", "Most Trophies", "Most Games")
```

Comment:
It might be a good solution to make an `enum` here for each tab category. So you don't need to test the equality of the `String` to know in which tab you are and maybe it's clearer and easier if you want to add more complex things to these tabs.

We could imagine that the enum has some attributes, for example: 
- The name of the tab
- The description of the tab (not necessary, but we can imagine for more complex tabs, if we want to add other tabs)
- A function that take a list of user and return a list of user ranked in a way (ranked by number of win)

We can also imagine that the list of users displayed would be limited to the first X users!